src/extra/widgets/chart/lv_chart.c

```diff
@@ -556,11 +555,8 @@ void lv_chart_set_next_value2(lv_obj_t * obj, lv_chart_series_t * ser, lv_coord_
 
     ser->x_points[ser->start_point] = x_value;
     ser->y_points[ser->start_point] = y_value;
-    invalidate_point(obj, ser->start_point);
```

Comment:
I removed this because this function is used only for scatter type, and inside the `invalidate_point` for the scatter type we invalidate the whole chart. 
I don't use scatter type, but I think it would be good to update only the needed area for this type too.